src/main/zip/start_dast_scan.groovy

```diff
@@ -15,6 +15,7 @@ final Properties props = airHelper.getStepProperties()
 
 String appId = props["applicationId"]
 String startingUrl = props["startingUrl"]
+String scanName = props["scanName"]?:startingUrl
```

Comment:
If you decide to change what I mentioned above, this is where I meant to trim the scan name when you do the check.